### PR TITLE
js91

### PR DESCRIPTION
**Pull request by satmandu**
- Add `js91`, add `gjs` & `polkit` built for `js91`.

Builds properly:
- [x] x86_64
- [x] armv7l
- [x] i686

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=js78 CREW_TESTING=1 crew update ; crew install js91 ; crew upgrade
```

### CODE REVIEW

File: packages/js78.rb [hunk 1]
```diff
@@ -49,13 +47,13 @@ def self.patch
     system('for i in `cat debian/patches/series`; do  patch -p 1 < debian/patches/$i; done')
```
Comment (supechicken) [inline packages/js78.rb:47]: Rubyize with `File.foreach 'debian/patches/series' {|i| system 'patch -p 1 < debian/patches/#{i}' }` ?

Comment (satmandu) [reply packages/js78.rb:47]: That gives this error:
```
js78.rb:47:42: E: Lint/Syntax: unexpected token tLCURLY
(Using Ruby 2.5 parser; configure using TargetRubyVersion parameter, under AllCops)
    File.foreach 'debian/patches/series' {|i| system 'patch -p 1 < debian/patches/#{i}' }
                                         ^
js78.rb:47:54: E: Lint/Syntax: unexpected token tSTRING
(Using Ruby 2.5 parser; configure using TargetRubyVersion parameter, under AllCops)
    File.foreach 'debian/patches/series' {|i| system 'patch -p 1 < debian/patches/#{i}' }
                                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
js78.rb:102:1: E: Lint/Syntax: unexpected token $end
(Using Ruby 2.5 parser; configure using TargetRubyVersion parameter, under AllCops)
```

Comment (supechicken) [reply packages/js78.rb:47]: Sorry, it should be `File.foreach 'debian/patches/series' {|i| system 'patch -p 1 < debian/patches/#{i}' }`

Comment (supechicken) [reply packages/js78.rb:47]: Sorry, it should be `File.foreach 'debian/patches/series' {|i| system "patch -p 1 < debian/patches/#{i}" }`

Comment (satmandu) [reply packages/js78.rb]: done

**Review by supechicken — APPROVED** [review]

**Comment by uberhacker** [comment]
Can we get an i686 build for this @satmandu?  The previous release had a working build.

**Comment by satmandu** [comment]
> Can we get an i686 build for this @satmandu? The previous release had a working build.

Once I have new hardware working and can confirm the build setup doesn't produce broken binaries...

File: packages/js78.rb (hunk 1 above)
Comment (satmandu) [reply packages/js78.rb]: Same error btw...

```
js78.rb:47:42: E: Lint/Syntax: unexpected token tLCURLY
(Using Ruby 2.5 parser; configure using TargetRubyVersion parameter, under AllCops)
    File.foreach 'debian/patches/series' {|i| system "patch -p 1 < debian/patches/#{i}" }
```

**Comment by satmandu** [comment]
@uberhacker This is good to go. js78 stays the same, but js91 gets added, and the packages which needed js78 get updated and rebuilt so that they point to js91.

**Comment by uberhacker** [comment]
Nice!

**Pull request merged**